.github/workflows/presubmit-script.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2025 The Android Open Source Project
```

Comment:
I think this should go into the existing repo-cecks rather than yet another run, otherwise it eats too much time for checkout and cache for no good reason